documentation/staging/content/userguide/prerequisites/introduction.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Operator prerequisites"
+date: 2019-02-23T16:40:54-05:00
+weight: 1
+---
+
+
+For the current production release 3.2.3:
+
+* Kubernetes 1.16.15+, 1.17.13+, 1.18.10+, and 1.19.7+ (check with `kubectl version`).
+* Flannel networking v0.9.1-amd64 or later (check with `docker images | grep flannel`), Calico networking v3.16.1 or later,
+ *or* OpenShift SDN on OpenShift 4.3 systems.
+* Docker 18.9.1 or 19.03.1+ (check with `docker version`) *or* CRI-O 1.14.7+ (check with `crictl version | grep RuntimeVersion`).
+* Helm 3.3.4+ (check with `helm version --client --short`).
+* For domain home source type `Model in Image`, WebLogic Deploy Tooling 1.9.11.
+* Either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930, Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0.
+   * The existing WebLogic Server image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`,
+   has all the necessary patches applied.
+   * Check the WLS version with `docker run container-registry.oracle.com/middleware/weblogic:12.2.1.3 sh -c` `'source $ORACLE_HOME/wlserver/server/bin/setWLSEnv.sh > /dev/null 2>&1 && java weblogic.version'`.
+   * Check the WLS patches with `docker run container-registry.oracle.com/middleware/weblogic:12.2.1.3 sh -c` `'$ORACLE_HOME/OPatch/opatch lspatches'`.
+* Container images based on Oracle Linux 8 are now supported. The Oracle Container Registry hosts container images
+  based on both Oracle Linux 7 and 8, including Oracle WebLogic Server 14.1.1.0.0 images based on Java 8 and 11.
+* You must have the `cluster-admin` role to install the operator.  The operator does
```

Comment:
1) --> You must have the Kubernetes `cluster-admin` ...
2) @rjeberhard : Is this still true if the operator runs in the same namespace as its domains?

Reply:
This is complicated. You don't need the specific `cluster-admin` role. Instead, you need sufficient roles that you have permission to create the CRD, cluster roles, and cluster role bindings. The `cluster-admin` role is the only built-in role that has sufficient privilege. You are, of course, making a different point, which (I believe) is that you don't need the `cluster-admin` role if you are installing the operator in the dedicated mode. That is correct. Someone else who does have the role would have needed to install the CRD in advance.

This should all be spelled out in the existing RBAC document. I wonder if we should just confirm that and then point to there from here?

Reply:
There is mention of the `dedicated` setting in the RBAC doc, https://oracle.github.io/weblogic-kubernetes-operator/security/rbac/. If it is sufficient, I will do as you suggest.

Reply:
The CRD is part of the operator - so I think the current wording is fine. We could add a 'See also' to the rbac doc here.

Reply:
Will do.